src/shared/components/palette/case-flag/components/manage-case-flags/manage-case-flags.component.ts

```diff
@@ -0,0 +1,109 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { ErrorMessage } from '../../../../../domain';
+import { CaseFlagState, FlagDetail, Flags } from '../../domain';
+import { CaseFlagFieldState, CaseFlagWizardStepTitle } from '../../enums';
+import { SelectManageCaseFlagErrorMessage } from '../../enums/manage-case-flag-type.enum';
+
+@Component({
+  selector: 'ccd-manage-case-flags',
+  templateUrl: './manage-case-flags.component.html'
+})
+export class ManageCaseFlagsComponent implements OnInit {
+
+  @Input() public formGroup: FormGroup;
+  @Input() public flagsData: Flags[];
+  @Output() public caseFlagStateEmitter: EventEmitter<CaseFlagState> = new EventEmitter<CaseFlagState>();
+
+  public flagLocationTitle: CaseFlagWizardStepTitle;
```

Comment:
@tarikdogan-hmcts - as already mentioned above, this should be changed to `flagSelectionTitle`